buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
```

Comment:
About the name & location of this file:

- it's named *.yml but other AWS config files are named *.yaml
- it's placed in the root dir but other AWS config files are placed in configuration dir

If there's no reason for the inconsistency, can you please make it consistent?